settings.gradle

```diff
@@ -1 +1,3 @@
 include ':app'
+include ':commons'
+project(":commons").projectDir = new File("/Users/darthpaul/StudioProjects/Simple-Commons/commons")
```

Comment:
guess this shouldnt be here. Try using some GUI for viewing changes before commiting